src/webview-ext.ios.ts

```diff
@@ -66,6 +66,8 @@ export class WebViewExt extends WebViewExtBase {
     }
 
     public disposeNativeView() {
+        this.wkWebViewConfiguration.userContentController.removeScriptMessageHandlerForName("nsBridge");
```

Comment:
Nice catch.

Just to make sure, should we call `this.wkWebViewConfiguration.userContentController.removeAllUserScripts()` too?

Reply:
Hm, I'm not sure. In my case it was not needed for the WKWebview to be cleaned up (it was enough with this patch).